package.json

```diff
@@ -123,6 +126,15 @@
       "scan": {
         "description": "[Scan]         Scan to get detailed security testing results: Visit https://saf.mitre.org/#/validate to explore and run inspec profiles"
       },
+      "supplement": {
```

Comment:
In the case of the comment below, does that mean that `supplement` alone would only essentially give you the help flag and then you call the examples below to actually run the command?

Reply:
Supplement by itself will output nothing, you need to choose one of either the passthrough or target sub(sub) commands to get anything to happen